cli/src/cli.rs

```diff
@@ -946,86 +943,42 @@ pub fn process_command(config: &CliConfig) -> ProcessResult {
             print_timestamp,
             compute_unit_price,
         } => {
-            let client_dyn: Arc<dyn TpsClient + 'static> = if config.use_tpu_client {
```

Comment:
We discussed a bit offline, and I mentioned in another comment, but let's create a normal async Option<TpuClient> here and pass that to ping to be used